### PR TITLE
ISS-634: Added/Updated Readme contents

### PR DESCRIPTION
**Pull request by sasmita**
Signed-off-by: Sisinty Sasmita Patra <sisinty.s.patra@intel.com>

### CODE REVIEW

File: google-pub/c/README.md [hunk 1]
```diff
@@ -1,55 +1,48 @@
...
+## Software requirements
+This version of the sample has been tested on Ubuntu Linux.
+
+## Setup
+Refer to [this document](https://cloud.google.com/iot/docs/quickstart) on how to create deviceid, registryid, and projectid parameters as they are used in calculating the client ID used for connecting to the MQTT bridge.
 
 # Building
-To build the sample, you must first download, build, and install the following
-projects:
+To build the sample, you must first download, build, and install the following projects, you can either install from individual links or follow the below commands.
```
Comment (anjgola) [inline google-pub/c/README.md]: Are these build steps are relevant to ISS?

Comment (anjgola) [inline google-pub/c/README.md]: Are these build steps are relevant to ISS? I believe all this should work in ISS right?
If so then lets have a section with 'Building outside ISS'  with these steps. 

Is there anything the user needs to do when building in ISS?

File: google-pub/c/README.md [hunk 2]
```diff
@@ -1,55 +1,48 @@
...
 
     export LD_LIBRARY_PATH=/usr/local/lib
     export LD_RUN_PATH=/usr/local/lib
 
-2. Browse to the sample folder
-
-    cd mqtt-ciotc
-
-3. Run the sample
+## Run the sample
```
Comment (anjgola) [inline google-pub/c/README.md]: The ISS readmes have header called 'SETUP'. Please show that add the argument value at project creation or added via the header files. 

Can the user provide command line arguments in ISS? If no, this should all go in a 'Building and Running sample outside ISS'

File: google-pub/c/README.md (hunk 1 above)
Comment (sasmita) [reply google-pub/c/README.md]: I see, I will keep all the building steps in a separate section as you mentioned. I don't think the user needs to do anything when building in ISS except the credentials it asks when running the sample.

File: google-pub/c/README.md (hunk 2 above)
Comment (sasmita) [reply google-pub/c/README.md]: I don't think user can provide command line arguments in ISS. I can add those steps in a. separate section.

**Comment by sasmita** [comment]
Thanks @anjgola for verifying. Sure, I will create a new JiRA item for that and regarding Readme, I will make sure to avoid "you". Just adding these steps from Quickstart guide as provided by Google, so kind of missed it. Do you want me to update again?

**Pull request merged**